doc/source/user_guide/launching_ansys_fluent.rst

```diff
@@ -1,7 +1,7 @@
 .. _ref_user_guide_launch:
 
-Launch Fluent locally
-=====================
+Launch or Connect to Fluent
```

Comment:
```suggestion
Launch or connect to Fluent
```
Changing to sentence-style capitalization (produced a Vale warning)

Reply:
Thanks